pom.xml

```diff
@@ -115,7 +154,7 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.8.1</version>
                 <configuration>
-                    <source>8</source>
+                    <source>12</source>
```

Comment:
Hm, really 12? Even having java 12 installed failed to compile with:
`[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile (default-compile) on project authenticator: Fatal error compiling: invalid source release: 12 -> [Help 1]`

`# java -version
java version "12.0.2" 2019-07-16
Java(TM) SE Runtime Environment (build 12.0.2+10)
Java HotSpot(TM) 64-Bit Server VM (build 12.0.2+10, mixed mode, sharing)
`